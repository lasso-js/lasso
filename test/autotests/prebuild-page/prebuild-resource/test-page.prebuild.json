[
  {
    "slots": {
      "body": "<script src=\"/static/test-page/lasso-modules-client$2.0.7/src/index.js\"></script>\n<script src=\"/static/test-page/lasso$3.4.3/test/autotests/prebuild-page/prebuild-resource/ebay.png.js\"></script>\n<script>$_mod.ready();</script>"
    },
    "assets": [],
    "name": "test-page"
  }
]