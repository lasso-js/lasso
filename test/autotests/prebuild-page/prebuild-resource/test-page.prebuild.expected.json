[
  {
    "slots": {
      "body": "<script src=\"/static/test-page/lasso-modules-client$2.0.5/src/index.js\"></script>\n<script src=\"/static/test-page/lasso$3.0.0-beta.10/test/autotests/prebuild-page/prebuild-resource/ebay.png.js\"></script>\n<script>$_mod.ready();</script>"
    },
    "assets": [],
    "name": "test-page"
  }
]
